packages/ibm-products/src/components/Datagrid/useRowIsMouseOver.js

```diff
@@ -29,7 +29,11 @@ const useRowIsMouseOver = (hooks) => {
       isMouseOver: row.index === mouseOverRowIndex,
     }));
 
-    Object.assign(instance, { rows: rowsWithMouseOver });
+    Object.assign(instance, {
+      rows: rowsWithMouseOver,
+      mouseHover: true,
```

Comment:
Can you rename this property? We've been prefixing values like this with `with`. For example, see [here](https://github.com/carbon-design-system/ibm-products/blob/main/packages/ibm-products/src/components/Datagrid/useStickyColumn.js#L103).

Usually we've just been replacing `use` from the name of the hook with `with`. In this case, `withRowIsMouseOver`.